auto_ts/__init__.py

```diff
@@ -136,10 +131,10 @@ def __init__(
         and Scikit-Learn ML. It will automatically select the BEST model which gives best score specified.
         #####################################################################################################
         """
-
+        lag_period = 5

```

Comment:
@AutoViML Why hard code this value? Should the user not be ale to decide this?

Reply:
At the moment: forecast_period is needed in model.predict() function. I felt that it did not belong in the auto_timeseries() object definition, hence I have just provided a placeholder value in object_definition. Will remove it once I find where all we have used forecast_period in the initialization object. I feel that it is not needed there. Once I find all the places it is used there and remove, then this lag_period will also be moved or removed. This is just a temporary situation. Hope this helps.

Reply:
Hi @AutoViML 

It is used to decide the cross validation steps (in `fit`), so it is needed. The idea is that you provide it once when you initialize the object and then you don't have to specify it again during `fit` or `predict`. 

The only time you need to provide it during `predict` is when you need the forecast horizon to be different than the one used for training (which is rare). I just added that extra flexibility in there in case some one needed it, but by default the  `forecast_period` is not needed to be specified in `predict`.

Could you please revert it back. If you are concerned that it needs a default value, you can add it to the `__init__` call and set it to 1 (instead of 5).

Thanks!